src/components/article/CodeBlock/CodeBlock.tsx

```diff
@@ -48,12 +49,13 @@ export const CodeBlock: VFC<Props> = ({
   editable = false,
   scope,
   withStyled = false,
+  renderingComponent = null,
   gap,
   align,
   layout,
 }) => {
   const language = className ? className.replace(/language-/, '') : ''
-  const code = children.trim()
+  const code = renderingComponent ? children.trim() + `\nrender(<${renderingComponent} />)` : children.trim()
```

Comment:
imo： 意図的でなければテンプレートリテラルの中に入れてしまいたい

```suggestion
  const code = renderingComponent ? `${children.trim()}\nrender(<${renderingComponent} />)` : children.trim()
```

Reply:
ここも、https://github.com/kufu/smarthr-design-system/pull/112/files#r894087096 と同様にコメントあると良いかもですね。tsxファイルをStorybookとreact-liveのプレビューとソースコード表示をよしなに両立するための複雑さをこの辺に押し込めているので。